analytical/utils.py

```diff
@@ -163,3 +163,56 @@ class AnalyticalException(Exception):
     be silenced in templates.
     """
     silent_variable_failure = True
+
+def build_paq_cmd(cmd, args=[]):
+    """
+    :Args:
+        - cmd -> The command to be pushed to paq (i.e enableHeartbeatTimer or contentInteraction)
+        - args -> Arguments to be added to the paq command. This is mainly
+            used when building commands to be used on manual event trigger.
+        
+    :Returns:
+        A complete '_paq.push([])' command in string form
+    """
+    paq = "_paq.push(['%s', " % (cmd)
+    if len(args) > 0:
+        for arg_idx in range(len(args)):
+            if arg_idx == len(args)-1:
+                paq += "'%s'])" % (args[arg_idx])
```

Comment:
Add a try/except to attempt and convert the value to a bool or int, if it's of either type then don't add the quotation marks